### PR TITLE
Fix: Added Launch Desktop button in Workspace[#2173] 

### PR DESCRIPTION
**Pull request by aakashreddy-p**
## Description
I have added a Launch Desktop button in the Workspace as well[#2173].

Please do not leave this blank 
## What type of PR is this? (check all applicable)
- [ ] 🐛 Bug Fix
## Have you tested locally?
- [ ] 👍 yes

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by aakashreddy-p** [comment]
> Please attach the issue link

https://github.com/actions/runner-images/issues/10636

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: apps/@sparrow-web/src/pages/workspace-page/CollectionsPage.svelte [hunk 1]
```diff
@@ -72,11 +73,13 @@
   import { SocketIORequestDefaultAliasBaseEnum } from "@sparrow/common/types/workspace/socket-io-request-base";
   import GraphqlExplorerPage from "./sub-pages/GraphqlExplorerPage/GraphqlExplorerPage.svelte";
   import { GraphqlRequestDefaultAliasBaseEnum } from "@sparrow/common/types/workspace/graphql-request-base";
+  import { TeamsViewModel } from "../Teams/Teams.ViewModel";
```
Comment (Astitva877) [inline apps/@sparrow-web/src/pages/workspace-page/CollectionsPage.svelte:76]: @aakashreddy-p  don't import teams view model here, in MVVM pattern we only import the view model with respective to it's page, so whatever functions you require add them in collectionview model and then use them.

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by aakashreddy-p** [comment]
> Please attach the issue link - https://github.com/actions/runner-images/issues/10636

**Review by Astitva877 — APPROVED** [review]

File: packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte [hunk 2]
```diff
@@ -716,6 +722,27 @@
         />
       </div>
     </div>
+
+    <!-- Launch sparrow desktop -->
+    <div class="px-3 pb-3">
```
Comment (Astitva877) [inline packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte:727]: @aakashreddy-p also add a condition that this button should be visible only in webapp not in desktop app

**Review by Astitva877 — CHANGES_REQUESTED** [review]

File: packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte (hunk 2 above)
Comment (aakashreddy-p) [reply packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte:727]: @Astitva877 okay I will add the condition in it.

Comment (aakashreddy-p) [reply packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte]: @Astitva877 I will fix it

File: packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte [hunk 3]
```diff
@@ -726,6 +732,29 @@
...
+            {#if windowOs}
+              <MacIcon height={"12px"} width={"12px"} color={"white"} />
+            {:else}
+              <WindowsIcon height={"12px"} width={"12px"} color={"white"} />
+            {/if}
+            <p class="ms-2 mb-0 text-fs-12" style="font-weight: 500;">
+              Launch Sparrow App
+            </p>
+          </div>
+        </button>
```
Comment (itsmdasifraza) [inline packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte:755]: convert this into an component and use in both teams and collection page

Comment (aakashreddy-p) [reply packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte:755]: I will create a component and I will add in the both.

File: packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte [hunk 4]
```diff
@@ -110,6 +113,10 @@
 
   export let testflows;
 
+  export let windowOs: boolean = false;
+
```
Comment (itsmdasifraza) [inline packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte:117]: not in use

File: packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte [hunk 5]
```diff
@@ -16,13 +16,15 @@
...
+    WindowsIcon,
     SocketIcon,
     TreeIcon,
     VectorIcon,
     BubbleIcon,
     StackIcon,
     SocketIoIcon,
     GraphIcon,
+    MacIcon,
   } from "@sparrow/library/icons";
```
Comment (itsmdasifraza) [inline packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte:28]: not in use

File: apps/@sparrow-web/src/pages/workspace-page/CollectionsPage.svelte [hunk 6]
```diff
@@ -419,6 +420,47 @@
...
+    let userAgent = window.navigator.userAgent;
+    if (userAgent.indexOf("Mac") != -1) {
+      windowOs = true;
+    } else if (userAgent.indexOf("Windows") != -1) {
+      windowOs = false;
+    }
+  }
+  onMount(() => {
+    getOS();
+  });
```
Comment (itsmdasifraza) [inline apps/@sparrow-web/src/pages/workspace-page/CollectionsPage.svelte:435]: not in use

File: packages/@sparrow-common/src/components/lanuch-desktop/LaunchSparrowAppDesktop.svelte [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    if (userAgent.indexOf("Mac") != -1) {
+      windowOs = true;
+    } else if (userAgent.indexOf("Windows") != -1) {
+      windowOs = false;
+    }
+  }
+  onMount(() => {
+    getOS();
+  });
+</script>
```
Comment (itsmdasifraza) [inline packages/@sparrow-common/src/components/lanuch-desktop/LaunchSparrowAppDesktop.svelte:17]: follow code patterns here
https://github.com/sparrowapp-dev/sparrow-app/blob/development/docs/CODE_PATTERNS.md

File: apps/@sparrow-web/src/pages/workspace-page/CollectionsPage.svelte [hunk 8]
```diff
@@ -419,6 +420,47 @@
...
+    // Check if app opened after a short delay
+    const detectAppTimeout = setTimeout(() => {
+      window.removeEventListener("blur", handleBlur);
+
+      // Only show popup if app wasn't detected
+      if (!appDetected) {
+        isPopupOpen = true;
+      }
+    }, 500);
+  }
```
Comment (itsmdasifraza) [inline apps/@sparrow-web/src/pages/workspace-page/CollectionsPage.svelte:463]: https://github.com/sparrowapp-dev/sparrow-app/blob/development/docs/CODE_PATTERNS.md

**Review by itsmdasifraza — APPROVED** [review]

**Review by itsmdasifraza — APPROVED** [review]

**Review by Astitva877 — APPROVED** [review]

**Pull request merged**